index.html

```diff
@@ -1,258 +1,322 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>mobile version</title>
-</head>
-<body>
-  <header class="header">
-      <a href="index.html">Arrow Dan</a>
-      <nav class="navbar container">
-        <img class="hamburger" src="image/Union.png" alt="navigation icon">
+  </head>
+
+  <body>
+    <header class="header">
+      <a class="container" href="index.html">Arrow Dan</a>
+      <nav class="navbar">
+        <img
+          class="hamburger hide-desktop"
+          src="image/Union.png"
+          alt="navigation icon"
+        />
+        <ul class="hide-mobile">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
       </nav>
-  </header>
+    </header>
 
-  <section class="main-container">
-    <div class="container">
-        <h1 class="logo-message">I'm  Dan</h1>
-        <h3 class="dev-intro">I'am a Software Developer</h3>
-        <p class="background">
-            I can help you build a product , feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded, do not hestiate to contact me.
-        </p>
-    </div>
-    <ul class="social-media  container">
-        <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
-        <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
-        <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
-        <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
-        <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
-    </ul>
-  </section>
+    <section class="main-container container">
+      <h1 class="logo-message">Hey there, I'm Dan</h1>
+      <h3 class="dev-intro">I'am a Software Developer</h3>
+      <p class="background">
```

Comment:
- [ ] Kindly place `I'm Dan` below `Hey there` as per the Figma design for both mobile and desktop versions.
- [ ] Kindly place the recent works in the centre for both mobile and desktop versions. When the size reduces it doesn't stay in the centre but move to one side.
- [ ] Kindly reduce the space between `Hey there` and `I'am Software Developer` as per the Figma design. Make sure to follow spacing according to the template chosen.
- [ ] Kindly follow the correct font family for `I'am Software Developer` for the desktop version.
- [ ] Kindly add the envelope icon to the navbar as per the Figma design to the header.
- Your Mobile design:
<img width="578" alt="Screenshot 2022-04-03 at 2 47 07 AM" src="https://user-images.githubusercontent.com/70269507/161405454-33067040-2a0c-4bc8-9c29-93b41b849605.png">

- Figma Mobile design:
<img width="260" alt="Screenshot 2022-04-03 at 2 55 02 AM" src="https://user-images.githubusercontent.com/70269507/161405412-9ea67ace-db32-420d-9d2e-c8a988cdb13a.png">

- Your Desktop design:
<img width="1649" alt="Screenshot 2022-04-03 at 2 58 37 AM" src="https://user-images.githubusercontent.com/70269507/161405486-7b2e64f4-3edf-488c-9daa-f902dfbf7a54.png">

- Figma desktop design:
<img width="574" alt="Screenshot 2022-04-03 at 2 55 37 AM" src="https://user-images.githubusercontent.com/70269507/161405417-de01036a-3954-46fd-bb9c-c44070dbcd15.png">